resources/assets/sass/patterns/_tables.scss

```diff
@@ -0,0 +1,44 @@
+/**
+ * Table
+ */
+.table-responsive {
+  width: 100%;
+  overflow-x: auto;
+}
+
+.table {
+  display: table;
+  width: 100%;
+  border: 1px solid $light-gray;
+  border-radius: 2px;
+}
+
+.table-header {
```

Comment:
Not to be _the worst_, but should this be `.table__header` since it's a child element of the `.table` pattern?